internal/pkg/deploy/cloudformation/stack/app.go

```diff
@@ -47,8 +47,8 @@ type AppRegionalResources struct {
 }
 
 const (
-	appTemplatePath               = "app/app.yml"
-	appResourcesTemplatePath      = "app/cf.yml"
+	appTemplatePath               = "app/versions/v1.0.0/app.yml"
+	appResourcesTemplatePath      = "app/versions/v1.0.0/cf.yml"
```

Comment:
Sorry for the follow comment 🤦 

Do you think we can create a similar constant in `deploy.LatestAppTemplateVersion` and `deploy.LatestAppInstanceTemplateVersion`? and use that instead of hardcoding the version here

Reply:
Thought about that. Will be added when we implement the `app upgrade`.